### DISCUSSION TITLE
DST not working?

### DISCUSSION THREAD

**Discussion by martinleo** [FossifyOrg/General-Discussion#634] [Q&A]
Hi, yesterday I set up my alarm for 8:30am. During the night the DST went live in Spain, but it appears FossifyClock ignored this and the alarm went off at 9:30am. I'm a little bit confused then, on whether this might be a bug or DST is not supported.
I'm on a P8P with GrapheneOS.
Thank you

**Comment by Aga-C** [comment]
It's a known issue from Simple Clock. Please raise a bug report for it.
